include/envoy/router/router.h

```diff
@@ -151,6 +151,7 @@ class RetryPolicy {
   static const uint32_t RETRY_ON_GRPC_DEADLINE_EXCEEDED  = 0x40;
   static const uint32_t RETRY_ON_GRPC_RESOURCE_EXHAUSTED = 0x80;
   static const uint32_t RETRY_ON_GRPC_UNAVAILABLE        = 0x100;
+  static const uint32_t RETRY_ON_GRPC_INTERNAL           = 0x200;
```

Comment:
This is going to conflict with your other change, so up to you how you want to sequence this.